test/nope_test.go

```diff
@@ -43,14 +43,21 @@ func TestMvMultiple(t *testing.T) {
 	testCommand("mv /this /should /be /multiple /commands", t)
 }
 
-func TestTouchSimple(t *testing.T) {
+func TestTouch(t *testing.T) {
 	var testFileName = "testfile.txt"
+	var rmCmd = "rm testfile.txt"
 
 	if _, err := os.Create(testFileName); err != nil {
 		t.Errorf("could not create file: " + testFileName)
 	}
 	var touchCmd = "touch " + testFileName
-	assert.Equal(t, testCommand(touchCmd, t), "rm testfile.txt", getTestPrompt("touch"))
+	assert.Equal(t, testCommand(touchCmd, t), rmCmd, getTestPrompt("touch"))
+
+	var touchCmdDontCreate = "touch -c whatever another"
+	assert.Equal(t, testCommand(touchCmdDontCreate, t), "", getTestPrompt("touch"))
+
+	var touchCmdSymLink = "touch -r dontdelete " + testFileName
+	assert.Equal(t, testCommand(touchCmdSymLink, t), rmCmd, getTestPrompt("touch"))
```

Comment:
Write test where error is thrown with incorrect flags. Should be something like assert Raises like in python. Not sure if that exists.